_rules/block-collapsible-3e12e1.md

```diff
@@ -0,0 +1,468 @@
+---
+id: 3e12e1
+name: Block of repeated content is collapsible
+rule_type: atomic
+description: |
+  This rule checks that repeated blocks of content are collapsible
+accessibility_requirements:
+  wcag-technique:SCR28: # Using an expandable and collapsible menu to bypass block of content
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - CSS Styling
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+  assets:
+    - _The Three Kingdoms_ by Luo Guanzhong, translation by Yu Sumei (Tuttle publishing, 2014, ISBN 9780804843935)
+---
+
+## Applicability
+
+This rule applies to any [HTML web page][].
+
+## Expectation
+
+For each [block of repeated content][] in each test target, which is before (in the [flat tree][]) at least one node of [perceivable content][] belonging to no [block of repeated content][], all the following are true:
+
+- there exists a [keyboard actionable][] [instrument][] to make all nodes in this [block][] not [visible][]; and
+- there exists an [instrument][], which is [included in the accessibility tree][], to remove all nodes in this [block][] from the [accessibility tree][included in the accessibility tree].
```

Comment:
🤔 
That note is actually part of the WCAG definition of "mechanism" which is used in the SC, so it's not just Understanding material but is part of the normative WCAG document, although only as a note rather than the main text…

---
So, there's one question of whether WCAG's "mechanism" must meet all SC at the same conformance level of the criterion is it used.

I would be at least a bit reluctant in saying that `<span  onclick="location.assign('#main');" aria-hidden="true">Click if you dare</span>` is sufficient to satisfy 2.4.1. It certainly is a mechanism to bypass block of repeated content, but it doesn't add much value…

---
And there's another question of whether we want the rule(s) to require that.

---
I'm seeing the two usual solutions:
- either we drop the requirement from the rule and add an Accessibility Support note saying that the rule is fairly lenient;
- or we keep the requirement in the rule with an assumption that this is part of the definition of "mechanism" (currently having an assumption for keyboard accessibility, would need one for inclusion in the accessibility tree).

I'm slightly more in favour of the second option given that the note on meeting other SC is part of WCAG's definition of "mechanism". But it's not a strong opinion.

---
Other rules that translate WCAG's mechanism (e.g. the single character shortcut one) do not have these requirements. I don't remember if this was discussed during the writing of the rule. But that leads me to another way to handle the question:
1. Locally remove these requirements from the rule. At least allows to ship the rule faster 🤞 
2. Open an Agenda item with the question since it is affecting several rules.
3. Depending on the outcome of the discussion, update our definition of "instrument" which is our translation of WCAG's mechanism (possibly simply adding an Accessibility Support note to it).